### PR TITLE
Separate db and model for Exposure/Publish

### PR DESCRIPTION
**Pull request by crwilcox**

### CODE REVIEW

File: internal/export/batcher.go [hunk 1]
```diff
@@ -25,6 +25,8 @@ import (
 	coredb "github.com/google/exposure-notifications-server/internal/database"
 	"github.com/google/exposure-notifications-server/internal/export/database"
 	"github.com/google/exposure-notifications-server/internal/export/model"
+	exposuremodel "github.com/google/exposure-notifications-server/internal/publish/model"
```
Comment (crwilcox) [inline internal/export/batcher.go:28]: I think we should go to either exposure everywhere or publish, for now I left it alone and try to stick to the way we were using it though.

Comment (sethvargo) [reply internal/export/batcher.go:28]: I like "publish" since all the other names are verbs, but either way I agree. This confused me a few times.

Comment (mikehelmick) [reply internal/export/batcher.go:28]: changing publish to exposure is fine -can be a follow up package rename only.

**Pull request merged**